transformer_engine/common/layer_norm/ln_api.cpp

```diff
@@ -235,20 +237,34 @@ void layernorm_fwd(const Tensor& x,        // BxSxhidden_size
         params.barrier = reinterpret_cast<int*>(barrier->data.dptr);
     }
 
+    const char *envval = std::getenv("NVTE_FORCE_MEMSET");
```

Comment:
The only nit is that it would be good to document the flag. This is a fall back to the original behavior of library.